src/utils/ethereum/chain.cljs

```diff
@@ -11,7 +11,9 @@
    :bsc         {:id   BSC-mainnet-chain-id
                  :name "BSC"}
    :bsc-testnet {:id   BSC-testnet-chain-id
-                 :name "BSC tetnet"}})
+                 :name "BSC tetnet"}
```

Comment:
there's a type -> `tetnet` te`s`tnet. but also why do we need this? we now get the network details from status-go and have this saved in a sub. so that it can be consistent with desktop etc

Reply:
Good catch. 
I don't know if we need this. I added the new chains here to stay consistent with what we already have. The spelling error has always been sneaking in the codebase. I can check for usages of this config. If there is none, I'll remove it.

Reply:
Cool, I actually think we can look to start deleting this file too as we are moving away from the old wallet and some of these chains. Outside the scope of this issue ofc

Reply:
I have fixed the typo. This variable is being used at multiple places, so it's not safe to remove yet.